data/core/doc/init.lua

```diff
@@ -34,6 +34,7 @@ function Doc:reset()
   self.lines = { "\n" }
   self.selections = { 1, 1, 1, 1 }
   self.cursor_clipboard = {}
+  self.cursor_clipboard_whole_line = {}
```

Comment:
What if we use `cursor_clipboard_whole_line` and `cursor_clipboard` globally (in `commands/doc.lua` maybe? or in `core`?) and not in each `doc`? This way one could also paste with multiple cursors in a different `doc`.

This way we also can remove `core.lines_in_clipboard` I think.

Reply:
Good ideas, I was a little tired when I was finishing it and made the simplest, but a bad solution.

Reply:
Do we need `full_text` at all?